database/migrations/2019_02_23_222617_add_hidpi.php

```diff
@@ -118,24 +100,6 @@ public function up()
 
 
 
```

Comment:
We need the following function to allow the migration to crash (in case pictures where not found).

Reply:
fixed